index.html

```diff
@@ -0,0 +1,242 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="menu.css">
+    <title>Document</title>
+</head>
+  <body>
+    
+    <nav class="toolbar">
+      <div class="logo"><a href="">My Logo</a></div>
+      <div class="menu_main">
+        <input type="checkbox" class="toggler">
+        <div class="humbuger"><div></div></div>
+        <div class="menu">
+          <div class="overlay">
+            <div>
+              <ul>
+                <li><a href="">Portifolio</a></li>
+                <li><a href="">About</a></li>
+                <li><a href="">Context</a></li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+
```

Comment:
[OPTIONAL] In accordance with Html semantic tag usage, I think it will be better to wrap your nav inside a `header` tag.😊